osu.Game/Skinning/SkinnableTargetWrapper.cs

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using Newtonsoft.Json;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+
+namespace osu.Game.Skinning
+{
+    /// <summary>
+    /// A container which is serialised and can encapsulate multiple skinnable elements into a single return type (for consumption via <see cref="ISkin.GetDrawableComponent"/>.
+    /// Will also optionally apply default cross-element layout dependencies when initialised from a non-deserialised source.
+    /// </summary>
```

Comment:
Rather than explaining that it's serialised, perhaps this should explain that this contains the elements of a `SkinnableTarget`? As far as I understand, that's what this is doing.

Reply:
Sure, open for rewording. Provide a suggestion to reduce back-and-forth if possible.

Reply:
How about

```suggestion
    /// <summary>
    /// A container which groups the elements of a <see cref="SkinnableTargetContainer"/> into a single object.
    /// Optionally also applies a default layout to the elements.
    /// </summary>
```